### ISSUE TITLE
Sublime Crash Startup Unable to find packages

### ISSUE THREAD

**Issue by molotovgod** [sublimehq/Packages#293]
Hello everyone,
I tried looking for an answer to my specific problem but I'm unable to find one. 
Latest version of Sublime 3 is crashing due to an infinite loop that I was doing for homework and ran it before finishing it.
If interested, here's the (simple, yet deadly) code using a Python build:
a=0
b=a+1
c=a+b
while c<5:
    print (c)
else:
    print ("loop will end")

I meant to add while c<5:
    print (str (c)+ "this loop will end)
    c=c+1
to print the iterations the loop takes to solve the problem.

The issue is that I believe sublime is stuck at an infinite loop and I'm unable to stop it neither with the keyboard nor the menu bar because I get the Mac's spinning colored wheel which prevents me from clicking anything.
I can also see my memory skyrocketing so I force-close the program. I tried uninstalling it completely and I've read that I can get rid of the session file to prevent from starting up with that code within ~library/Application Support/Sublime/ etc, however this folder doesn't show up within the Application Support folder. I've tried rolling back to other versions and it just keeps loading the code again, I'm unable to find a solution for this. Would anyone care to help, please?
Thank you in advance!
-Ivan

**Comment by jfcherng** [comment]
Are you finding a way to prevent from hanging if you accidentally write a infinite loop?

**Comment by FichteFoll** [comment]
They key issue here isn't just that you have an infinite loop but rather that the process you ran pipes data into stdout indefinitely and Sublime Text can't keep up. A way to get out of this situation is to kill the process ST spawned by running the build system.

There is no way for ST to recognize whether an application is emitting data endlessly or just a lot of it because the halting problem is unsolvable. It would be nice if it still responded to user events however,

**Comment by jfcherng** [comment]
An workaround could be execute your program in an external terminal.
For example, using [Console Exec](https://packagecontrol.io/packages/Console%20Exec).

**Comment by wbond** [comment]
This doesn't seem to be a bug with the default packages.

**Comment by molotovgod** [comment]
Hello everyone, 
I was truly confused but I managed to fix it. 
Thank you for your input however. 
I never realized that in a Mac there are two Library folders, one for the main drive and one for the user folder, which had the contents under folder Application Support folder but I had to unhide all files to see it under user.

Show all files on mac from terminal:

```
defaults write com.apple.finder AppleShowAllFiles TRUE

killall Finder
```
Navigate to your User folder, once Library shows open in new tab with finder and find ~Library/Application Support/Sublime Text 3/...  delete all sessions for Sublime Text.
Hide all files and done
`defaults write com.apple.finder AppleShowAllFiles TRUE

killall Finder`
